semgrep-core/src/targeting/Skip_target.ml

```diff
@@ -23,6 +23,13 @@ module Resp = Output_from_core_t
      than 500 bytes at the same time.
    - only some really short source files (2-3 lines) were found to have
      between 5% and 8% whitespace.
+
+  However, this threshold works less well for java files because it is not
+  unusual to have large import blocks at the top, resulting in a very low
+  whitespace fraction. This should not affect very many Java files, so for
+  now it is ok that we leave it. The consequence of reducing the frequency
+  is that Semgrep would likely take longer as it scans more costly minified
+  files. TODO we might want to do some benchmarking and change this default
```

Comment:
Could we also tweak this default so that it works for your example?

Reply:
What was the `stat.ws_freq` for that example?

Reply:
Yeah, we could do that. I wanted to just not skip them all for now because they might be helpful and we aren't focused on performance yet.

Reply:
From my example I would have lowered it to 0.3